### ISSUE TITLE
Support for v1.2 adaptive cards in Microsoft teams

### ISSUE THREAD

**Issue by gturbahn1** [microsoft/AdaptiveCards#3715]
Since no one is responding to the #3480 issue any more, I'm opening a new one.

We've been waiting on support for Adaptive Cards 1.2 on Microsoft Teams for a while now, but haven't seen any update.

Could you please provide an update on this?

Thanks!

**Comment by shalinijoshi19** [comment]
Apologies for the tardiness here @gturbahn1 ; Responded in #3480 . Closing this one out as duplicate of #3480
